tests/main.cpp

```diff
@@ -788,3 +788,56 @@ TEST_CASE("getEpochSchedule") {
       epochschedule2.getLastSlotInEpoch(16),
       epochschedule2.firstNormalSlot + 3 * epochschedule2.slotsPerEpoch - 1);
 }
+
+TEST_CASE("stakeactivation") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto stakeactivation =
+      connection.getStakeActivation(solana::PublicKey::fromBase58(
+          "CYRJWqiSjLitBAcRxPvWpgX3s5TvmN2SuRY3eEYypFvT"));
+  CHECK_GE(stakeactivation.active, 0);
+  CHECK_GE(stakeactivation.inactive, 0);
+  CHECK((stakeactivation.state == "active" ||
+         stakeactivation.state == "inactive" ||
+         stakeactivation.state == "activating" ||
+         stakeactivation.state == "deactivating"));
+}
+
+TEST_CASE("getInflationGovernor") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto inflationgovernor = connection.getInflationGovernor(
+      solana::commitmentconfig{solana::Commitment::FINALIZED});
+  CHECK_GE(inflationgovernor.foundation, 0);
+  CHECK_GE(inflationgovernor.foundationTerm, 0);
+  CHECK_GE(inflationgovernor.initial, 0);
+  CHECK_GE(inflationgovernor.taper, 0);
+  CHECK_GE(inflationgovernor.terminal, 0);
+}
+
+TEST_CASE("getTransactionCount") {
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto TransactionCount = connection.getTransactionCount(
+      solana::GetSlotConfig{solana::Commitment::FINALIZED});
+  CHECK_GT(TransactionCount, 0);
+  sleep(2);
```

Comment:
May be create some transactions confirm them and test again instead of waiting for 2 s.

Reply:
Done